test/e2e/tests/tokens/custom-token-send-transfer.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { Mockttp } from 'mockttp';
-import { mockedSourcifyTokenSend } from '../confirmations/transactions/erc20-token-send-redesign.spec';
```

Comment:
this spec import, caused to run all the specs in the `erc20-token-send-redesign.spec` file, and subsequently the specs in the current file `custom-token-send-transfer` in the same context, making those fail as sometimes that takes more than 80000ms

Reply:
great catch  !